test/C/src/ArrayAsParameter.lf

```diff
@@ -3,9 +3,9 @@
 // encode their own length.
 target C
 
-reactor Source(sequence: int[](0, 1, 2), n_sequence: int(3)) {
+reactor Source(sequence: int[] = { 0, 1, 2 }, n_sequence: int = 3) {
```

Comment:
This looks so much cleaner! I only find the extra spaces after `{` and before `}` a bit odd.